dev-setup/dev-setup.yml

```diff
@@ -0,0 +1,22 @@
+---
+- hosts: localhost
+  become: true
+  roles:
+    - geerlingguy.docker
+  tasks:
+    - name: install python-pip
```

Comment:
Python Pip and CQLSH aren't needed anymore, since we're using cqlsh directly from the container. Please remove.